### DISCUSSION TITLE
pnpm instead of npm

### DISCUSSION THREAD

**Discussion by peterbud** [Azure/azure-dev#3018] [Q&A]
Is it possible / are there any plans to support `pnpm` instead of `npm` with `azd`?

As I see in the source npm is a hardcoded service, and I'm wondering whether there is an option, or plan to add or support pnpm?

**Comment by rajeshkamal5050** [comment]
Tracked via https://github.com/Azure/azure-dev/issues/4372

**Comment by jongio** [comment]
This has been resolved with https://github.com/Azure/azure-dev/pull/6894, which adds support for configurable package managers including pnpm.
